src/core/src/bootstrap/Constants.py

```diff
@@ -98,6 +98,33 @@ class ConfigSettings(EnumBackport):
     # patch versions for healthstore when there is no maintenance run id
     PATCH_VERSION_UNKNOWN = "UNKNOWN"
 
+    # Strings used in perf logs
+    class LogStrings:
+        TASK = "task"
+        TASK_STATUS = "task_status"
+        ASSESSMENT = "Assessment"
+        PATCH_INSTALLATION = "PatchInstallation"
```

Comment:
We do have operation constants already defined here (see lines 87-92, under #Operations), should re-use those

Reply:
Done. Reusing the constant strings for Assessment and Installation